.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+Short description explaining the high-level reason for the new issue.
+
+## Current behavior
+
+-
+
+## Expected behavior
+
+-
+
+## Steps to replicate behavior
+
+1.
+
+
+## Screenshots
+
+__Current__
+
+
+__Expected__
```

Comment:
I was torn on this, because I always want to see screenshots if possible, and designers are one of our top issue posters, but it also doesn't need to be required. Maybe I should add a note at the top that says to fill out all that's applicable?